platform/ios/src/MGLMapView.h

```diff
@@ -653,6 +657,9 @@ MGL_EXPORT IB_DESIGNABLE
  @param animated Specify `YES` if you want the map view to animate scrolling and
     zooming to the new location or `NO` if you want the map to display the new
     location immediately.
+ 
+ @note The behaviour of this method is undefined if called when observing
```

Comment:
Nit: “when observing” sounds like it applies at any time after registering for the notification, which could be throughout the lifetime of the application. “In response to” would be clearer.

Reply:
Will change.